modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartClientCodegen.java

```diff
@@ -39,16 +39,22 @@
     public static final String PUB_NAME = "pubName";
     public static final String PUB_VERSION = "pubVersion";
     public static final String PUB_DESCRIPTION = "pubDescription";
+    public static final String PUB_AUTHOR = "pubAuthor";
+    public static final String PUB_AUTHOR_EMAIL = "pubAuthorEmail";
+    public static final String PUB_HOMEPAGE = "pubHomepage";
```

Comment:
What about using the following as the default value instead?

```
    public static final String PUB_AUTHOR = "OpenAPI Generator Contributors";
    public static final String PUB_AUTHOR_EMAIL = "team@openapitools.org";
    public static final String PUB_HOMEPAGE = "https://openapi-generator.tech";
```

Reply:
For the author this could be done, because the publisher would have to change this before uploading since the author's email used to authorize further updates of the package

For the homepage I wanted to prevent that the publisher uploads a dart package which links to openapi and thus indicating that the different projects are somehow linked
If that's no problem from your point of view I will change it.

Reply:
What do you think about setting all of the PUB_ default values (except PUB_NAME) to null and then not adding anything to the pubspec when the value is null?  I'm wondering if nothing is better than a dummy value that might be confusing or problematic.  What do you both think?

Reply:
would be a possibility, everything except PUB_NAME is required only for packages which are published to pub